From 6cf335aa30de0af25ea82b2f21e4a6cd82f7747e Mon Sep 17 00:00:00 2001
From: Sugar Zhang <sugar.zhang@rock-chips.com>
Date: Tue, 10 Nov 2020 21:42:34 +0800
Subject: [PATCH] arm64: configs: rockchip: Enable digital codec interface

Digital codec interface is used with external codec analog part
with pdm link(e.g.: rk812) or audiopwm out differential.

Change-Id: Ic649cf1861cdcd493d52176a627ba7bbdf3338ae
Signed-off-by: Sugar Zhang <sugar.zhang@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 96ec87e0b568..6475f8162852 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -633,6 +633,7 @@ CONFIG_SND_SOC_ES8316=y
 CONFIG_SND_SOC_ES8396=y
 CONFIG_SND_SOC_RK3328=y
 CONFIG_SND_SOC_RK817=y
+CONFIG_SND_SOC_RK_CODEC_DIGITAL=y
 CONFIG_SND_SOC_RT5640=y
 CONFIG_SND_SOC_SPDIF=y
 CONFIG_SND_SIMPLE_CARD=y
-- 
2.35.3

